.github/workflows/linting.yml

```diff
@@ -58,3 +58,12 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Check DAOS logging macro use.
         run: ./utils/cq/d_logging_check.py --github src
+
+  ftest-tags:
+    name: Ftest tag check
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Check DAOS ftest tags.
+        run: \[ ! -x src/tests/ftest/tags.py \] || ./src/tests/ftest/tags.py lint
```

Comment:
If however you're not doing annotations then you can leave the checkout code as it is and it'll checkout a merge commit and therefore this file should always exist and therefore you don't need this protection.

Reply:
Thanks. I think ultimately we want to annotate file names, but not source lines